fe/fe-core/src/main/java/org/apache/doris/nereids/memo/GroupExpression.java

```diff
@@ -49,6 +49,8 @@ public class GroupExpression {
     private final BitSet ruleMasks;
     private boolean statDerived;
 
+    private long estOutputRowCount = -1;
```

Comment:
why add this to group expression? we could get row count from group

Reply:
this is for debug purpose. different groupExpression may have different estimated row count.